src/test/scala/com/redhat/et/silex/statistics/cramsersv.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * cramersv.scala
+ * author:  RJ Nowling <rnowling@redhat.com>
+ *
+ * Copyright (c) 2016 Red Hat, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.redhat.et.silex.statistics
+
+import com.redhat.et.silex.testing.PerTestSparkContext
+
+import org.scalatest._
+
+class CramersVSpec extends FlatSpec with Matchers with PerTestSparkContext {
+
+  val eps = 1e-5
+
+  "CramersV.cramersV" should "report 1.0 under perfect association" in {
```

Comment:
This is a nit, but the test names should be updated since there isn't a `cramersV` method any more